modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordApi.kt

```diff
@@ -546,6 +555,20 @@ class MirrordApi(private val service: MirrordProjectService, private val project
                 service.notifier.notification("For more features of mirrord, including multi-pod impersonation, check out mirrord for Teams!", NotificationType.INFORMATION).withLink("Try it now", MIRRORD_FOR_TEAMS_URL).withDontShowAgain(MirrordSettingsState.NotificationId.MIRRORD_FOR_TEAMS).fire()
             }
         }
+
+        if (currentRuns == NEWSLETTER_INVITE_AFTER_FIRST || currentRuns == NEWSLETTER_INVITE_AFTER_SECOND || currentRuns == NEWSLETTER_INVITE_AFTER_THIRD) {
```

Comment:
Can't we do the `when` instead of this `if`? Some sort of `when(currentRuns) { blah blah }.then { print it }`?